Automattic/wp-calypso
# [Odie] Kick-off: Odie integration into Help Center

This is the initial set of changes to replace or "revamp" the `Help Center` with AI capabilities from `ai-services` & `Odie`

## Proposed Changes

There are a bunch of changes that are meant to make possible the old `Help Center` and the revamped one to be co-existing at the same time. To make it possible, we've added a new flag, which is `wapuu`

The initial design is creating a simple approach taking as an initial design the one proposed by Andrew here: peGwbA-EJ-p2#comment-1209

## Testing Instructions

Is very important to test the regular flow for the Help Center. These set of changes should not be visible if the feature flag is not enabled.

1. Open the Live link
2. Test in depth the `Help Center`
    1. Check that clicking on the `Help Center` icon in the `Master Bar` is showing/hiding the `Help Center`
    2. Check that the navigation within the `Help Center` is still working, eg. You can search for docs, you are still able to contact us, you are still able to go back and forth in the navigation and any button still working.
3. Enable the new `Help Center` adding to the URL: `?flags=wapuu`
4. Check that the UI on the new `Help Center` has changed as it will show a new option in Still Need Help -> Wapuu.
5. Wapuu is not able to reply yet. The presented screen is just a mock and is not connected to our backend services.
6. Test that the Help Center is still able to be minimised, dragged, closed, and that it keeps the state while minimised.

eg:
https://github.com/Automattic/wp-calypso/assets/5689927/cb759aed-7468-4e36-bc91-fef1362bd60f

## Pre-merge Checklist
